docs/content/Upgrade/automatedupgrade.md

```diff
@@ -120,6 +120,20 @@ Previously, the existing cluster upgrade focused on updating a cluster's underly
 
 The automated upgrade process provides a mechanism where, instead of being deleted, the existing PostgreSQL clusters will be left in place during the PostgreSQL Operator upgrade. While normal Operator functionality will be restricted on these existing clusters until they are upgraded to the currently installed PostgreSQL Operator version, the pods, services, etc will still be in place and accessible via other methods (e.g. kubectl, service IP, etc).
 
+
+{{% notice warning %}}Automated upgrade process does not support upgrading clusters that have not been initialized. Ensure that the following status is set on your `pgcluster` before attempting the automated upgrade:
```

Comment:
:pick: Just wondering if it might sound better to say "The automated upgrade process" (just added a "the" in front).  Also, can we say "have not been fully initialized" (added "fully").  So maybe:

> The automated upgrade process does not support upgrading clusters that have not been fully initialized (note that full cluster initialization includes successful pgBackRest stanza creation, as well as a successful initial backup).  Therefore,  ensure that the following status is set on your `pgcluster` before attempting the automated upgrade: